Tools/LambdaTestTool/src/Amazon.Lambda.TestTool.WebTester21/Controllers/TesterController.cs

```diff
@@ -2,6 +2,7 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
+using System.Text.Json;
```

Comment:
Since we already have a dependency on Newtonsoft we should use that instead of pulling in the System.Text.Json dependency for .NET Core 2.1. The more dependencies we take on here the more likely we will collide with the dependencies used by the Lambda function.